src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="iMac"
+        class="product-card__img"
+      >
+      <div class="product-card__title">
+        Моноблок APPLE A1419 iMac <br>
+        27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="product-card__code">
+        Код товара: 195434
+      </div>
```

Comment:
```suggestion
      <h3 class="product-card__title">
        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
      </h3>
      <span class="product-card__code">
        Код товара: 195434
      </span>
```
Let's use more semantic tags instead of `div`